### PR TITLE
FAQ

### LINKED ISSUE #229: FAQ section
It will be useful to have a section with "frequently asked questions" in the handbook. Questions that are covered in the book (such as: "How can I delete a dataset?") can link to appropriate sections, questions that are not part of a course (such as: "What is the difference between gitfs and DataLad?"

Questions to cover:
- What differentiates DataLad from gitfs and flywheel?
- How much is actually saved?

[please extend]

### PR DESCRIPTION
**Pull request by adswa**
First bunch of FAQs.
There are still many TODOs, and this section will be added to frequently. After review, I'd be fine with leaving this PR open until more FAQs are collected and answered, or to merge and frequently update this section.

### TODO
- how do I apply a configuration change (e.g., place all text files in Git) to files that were handled with previous defaults (i.e., I already have a bunch of text files that are annexed) ?
- [x] Does DataLad host my data?
- [x] Is there a graphical user interface to DataLad?
- [x] Python 2 support

This fixes #229.

### CODE REVIEW

File: docs/basics/101-180-FAQ.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+A more elaborate delineation from related solutions can be found in the DataLad
+`developer documentation <http://docs.datalad.org/en/latest/related.html>`_.
+
+DataLad version-controls my large files -- great. But how much is saved in total?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+   this.
```
Comment (mih) [inline docs/basics/101-180-FAQ.rst]: "saved" as in duplicates removed, or "saved" as in "tracked"?

File: docs/basics/101-180-FAQ.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+How can I copy data out of a DataLad dataset?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Moving or copying data out of a DataLad dataset is always possible and works in
+many cases just like in any regular directory. The only
+caveat exists in the case of annexed data: If file content is managed with
+Git-annex and stored in the :term:`object-tree`, what *appears* to be the
+file in the dataset is merely a symlink (please read section :ref:`symlink`
+to understand this). Moving or copying this symlink will not yield the
```
Comment (mih) [inline docs/basics/101-180-FAQ.rst]: "to understand this" -> "for details"

File: docs/basics/101-180-FAQ.rst [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+intended result -- instead you will have a broken symlink outside of your
+dataset.
+
+When using the terminal command ``cp``, it is sufficient to use the
+``-L``/``--dereference`` option. This will follow symbolic links, and make
+sure that content gets moved instead of symlinks.
+With tools other than ``cp`` (e.g., graphical file managers), to copy or move
```
Comment (mih) [inline docs/basics/101-180-FAQ.rst]: The absolutely amazing Midnight Commander `mc` can also follow symlinks
image

**Comment by adswa** [comment]
Thx for the comments!

**Comment by mih** [comment]
Think is good. Further work can come later in a separate PR. Merging. Thx!

**Pull request merged**